h/accounts/schemas.py

```diff
@@ -102,6 +110,31 @@ def new_password_node(**kwargs):
         **kwargs)
 
 
+def _privacy_accepted_message():
+    terms_links = {
```

Comment:
I think you can make a jinja extension or a macro for these?

Reply:
Hmmm...I think it could be pretty complex as the rendering of the form in the page template is entirely encapsulated within `{{ form }}` which is the rendered form. I'm not sure where I'd be able to, like, inject that macro.